netcore/Makefile.am

```diff
@@ -73,13 +66,15 @@ clean:
 # with rules like tes-Foo which runs limited set of test
 #
 
-# e.g. `make xtest-System.Collections.Tests COREFX_ROOT=/prj/corefx`
+# e.g. `make xtest-System.Collections.Tests`
 # '-parallel none -verbose' for debug, `-parallel all` to run all tests in parallel
-# we need COREFX_ROOT only for xunit.console.deps.json and CoreFx.Private.TestUtilities.dll
 xtest-%: prepare check-env update-corefx-tests
```

Comment:
`check-env` should be removed then (as a dependency for xtest* rules)

Reply:
Oops, I forgot to do that.